flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/source/TestStreamScanSql.java

```diff
@@ -30,6 +30,7 @@
 import org.apache.flink.table.api.config.TableConfigOptions;
 import org.apache.flink.types.Row;
 import org.apache.flink.util.CloseableIterator;
+import org.apache.iceberg.AssertHelpers;
```

Comment:
This makes me realized that we don't have the stream sql scan for the FLIP-27 source.

Reply:
added one in TestStreamingMonitorFunction.